js/xena.js

```diff
@@ -10,7 +10,17 @@ module.exports = class xena extends Exchange {
             'id': 'xena',
             'name': 'Xena Exchange',
             'countries': [ 'VC', 'UK' ],
-            'rateLimit': 100,
+            // per second rate limits are far lower than the equivalent hourly
+            // requests per second rounded down (3dp)
+            // relative weight costs rounded up (3dp)
+            // 1 hour = 3600 seconds
+            // Order Cancellations: 100k per hour => 100 000 / 3600 = 27.777 requests per second => rateLimit = 1000ms / 27.777 = 36.001008 ms between requests => 36.1 (safety)
+            // New Orders: 30k per hour => 30 000 / 3600 = 8.333 requests per second => cost = 27.777 / 8.333 = 3.333373335 => 3.334
+            // Heartbeat: 30k per hour => 30 000 / 3600 = 8.333 requests per second => cost = 27.777 / 8.333 = 3.333373335 => 3.334
+            // Candles: 5000 per hour => 5000 /  3600 = 1.388 requests per second => cost = 27.777 / 1.388 = 20.01224784 => 20.013
+            // Dom (market data): 5000 per hour => 5000 /  3600 = 1.388 requests per second => cost = 27.777 / 1.388 = 20.01224784 => 20.013
+            // All snapshot requests (balances, active orders and trade history, positions): 500 per hour => 0.138 requests per second => cost = 27.777 / 0.138 = 201.2826087 => 201.283
```

Comment:
Are we sure about 201.283 ? Sounds like a very harsh limit, roughly once every 10 seconds, which isn't very convenient for bots. Does it work with any lower values?

Reply:
I can try lower values, but this is the 'hourly' limit, I tried the per second rate limit -  and got ratelimited in about 10 minutes or so 

Xena Rate Limits:  https://support.xena.exchange/support/solutions/folders/44001210111

I'll try something in between, but i think that it will end up hitting errors after running for close to an hour

Reply:
@DoctorSlimm let me know your results, i'll merge when ready.

Reply:
works with the pushed lower weights

Reply:
how did you decide those rates? I think we should use the second ones

Reply:
we shouldn't randomly decide weights imo

Reply:
@frosty00 Which second ones you mean? Igor said those limits are too high (the ones stated in the docs for the private get endpoints) and I tested with lower ones than stated and he was correct that they did work actually, despite being stated to be much higher in the docs

Reply:
@DoctorSlimm i think @frosty00 is asking why we set a cost of 50, specifically, and not 25, nor any other number. If it worked with 50 and didn't work with 49, that would explain it. Otherwise, we might have to either find the lowest working value or revert to the numbers from their API docs.